File: src/java/HelloWorld/src/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import android.os.Bundle;
import android.app.Activity;
import android.graphics.Color;
import android.view.Menu;
import android.view.ViewGroup.LayoutParams;
import android.widget.LinearLayout;
import android.widget.RelativeLayout;
import android.widget.TextView;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        
        RelativeLayout layout = (RelativeLayout)findViewById(R.id.layout);
        
        TextView textView = new TextView(this);
        
        textView.setText("Hello TechEd 2013!");
        textView.setTextColor(Color.RED);
        
        LayoutParams layoutParams = new LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT);
        
        layoutParams.height = 30;
        layoutParams.width = 200;
  
		textView.setLayoutParams(layoutParams);
        layout.addView(textView);
        
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.main, menu);
        return true;
    }
    
}

File: src/cs/HelloWorld/HelloWorld.Android/MainActivity.cs
using Android.App;
using Android.Views;
using Android.Widget;
using Android.OS;
using Android.Graphics;

namespace HelloWorld.Android
{
	[Activity (Label = "Hello World", MainLauncher = true)]
	public class Activity1 : Activity
	{
		protected override void OnCreate (Bundle bundle)
		{
			base.OnCreate (bundle);

			SetContentView (Resource.Layout.Main);

			var layout = (RelativeLayout)FindViewById(Resource.Id.layout);

			var layoutParams = new ViewGroup.LayoutParams(
                ViewGroup.LayoutParams.WrapContent, 
                ViewGroup.LayoutParams.WrapContent)
			    {
			        Width = 300,
			        Height = 30
			    };

		    var textView = new TextView(this)
		        {
		            Text = "Hello TechEd 2013",
                    LayoutParameters = layoutParams
		        };

		    textView.SetTextColor(Color.Red);

			layout.AddView(textView);
		}
	}
}


